crates/core/tedge_mapper/src/c8y/mapper.rs

```diff
@@ -72,7 +74,36 @@ impl TEdgeComponent for CumulocityMapper {
                 tc.forward_from_remote(topic, local_prefix.clone(), "")?;
             }
 
-            tc.forward_from_local("#", local_prefix, "")?;
```

Comment:
I don't think this was genuinely problematic as Cumulocity doesn't send us our own messages back, but I've changed the config to match the current bridge config in any case